_service-details/secchamps.md

```diff
@@ -0,0 +1,62 @@
+
+That's where Bounce Security comes in.
+
+We use our significant experience in application security as well as our long experience of working directly with developers and within developement organizations to help you build and operate a security champions programme that works for your organization.
+
+We have seen enough Security Champion Programmes in the past to understand well the practices that makes a programme successful and anti-patterns that may reduce the programme's efficiency or even cause it to fail.
+
+With our focus on activities that bring value and impact, the outcome will be interested and engaged developers who are incentivised to excel and provided the resources and support to do so. In turn, this will lead to an improved application security culture within the organization and less security effort required later on in the process.
+
+Our "Managed Security Champions Programme" service is split into a set of options that can be mixed and matched according to what will bring you the highest value right now.
+
+### Build
+
+#### Making the Case
+
+You want a programme but you need internal buy-in? We can help you identify the relevant stakeholders, prepare advantages specific to your organization and get everyone on board.
+
+#### Green-field Design
+
+You don't yet have a champions programme? We will review your organization's structure and provide a suggested template for starting up the programme and expected roles for champions.
+
+#### Build the programme
+
+You already have a template or you know what is needed? At your direction, we will help you set up the programme including initial communications, internal marketing and activity structure.
+
+### Revitalize
+
+#### Current State Analysis
+
+You have a programme but you want to know how it's going? We will compare your current programme to leading practices in the industry and identify where you are doing well and where you might have gaps.
+
+#### Improvement Plan
+
+You want to take your Security Champions Programme to the next level? We will prepare a phased plan to enhance and improve your existing programme, with a focus on getting better value.
+
+#### Level up your Champions
+
+Your programme is working well and your champions are hungry for more? We can help you build a staged developement programme for them to take them through various levels of expertise so that they always have something to strive for and something new to learn.
+
+### Operate
+
+#### Ongoing Communication
+
+Your champions need ongoing updates and information about what is going on in the industry, but preparing that is time consuming and needs constant work. We will prepare a monthly session for your champions including industry news, a more in-depth topic and your internal updates which you provide us with. We will also encourage your champions to be involved and deliver their own topics to improve their engagement and buy-in.

```

Comment:
re internal updates: the last part makes this seem unenticing. Perhaps something more like "... and internal updates which we'll prepare together."
or even just "internal updates" and leave it at that.

Reply:
Yeah I said `... and internal updates which we'll prepare together.`

There is always some input we will need from them

Reply:
can we just leave it as "internal updates"? It feels like it leaves off on a bad taste. And its not like its contractually binding...

Reply:
Done